libcontainer/state_linux.go

```diff
@@ -0,0 +1,206 @@
+// +build linux
```

Comment:
This file is not containing the Checkpoint transition, was it intentional ?
if it was intentional, do we need the state Checkpoint in "container interface" ? Even in refreshState function I'm not seeing the change ?
Not sure whether it was intentional change you made